docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -77,6 +77,65 @@ At startup, this connector tries guessing fields' types, but it might not be cor
 
 This property is optional; the default is ``_schema``.
 
+.. note::
+
+    Trino can autogenerate the ``_schema`` contents by running a ``SELECT *`` over the collection.
+    However, not all the fields are always detected and you might need to update the schema manually or using ``ALTER TABLE``.
```

Comment:
What is the difference between updating the schema manually and using ALTER TABLE? Might want to explain.

Reply:
With manually I meant updating directly on MongoDB, and `ALTER TABLE` is through Trino.

Reply:
Great. Should be explicit about that for our users.

Reply:
I updated it to be more explicit about the change.